### PR TITLE
fix(DDI-1031): Remove margin-bottom from last element in content area

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Removed the margin-bottom from last element in modal-content

### CODE REVIEW

File: libs/web-components/src/components/modal/Modal.svelte [hunk 1]
```diff
@@ -199,6 +199,11 @@
     margin: 1.5rem 0 0;
   }
 
+  .modal-content ::slotted(:last-child) {
+    margin-bottom: unset !important;
```
Comment (chrisolsen) [inline libs/web-components/src/components/modal/Modal.svelte:203]: Is there a reason this is not being set to 0?

Comment (Radha-Alberta) [reply libs/web-components/src/components/modal/Modal.svelte:203]: Changed the margin-bottom to 0

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**